### DISCUSSION TITLE
Modbus Error: Failed to connect

### DISCUSSION THREAD

**Discussion by yazansoofi** [artisan-roaster-scope/artisan#1058] [General]
Hello all, i am new to artisan, i downloaded artisan-win-2.8.2  for my windows computer. i am roasting on Diedrich Dr-3. i can not connect the roaster with artisan. i keep receiving Modbus error: Failed to connect. Any suggestions would be highly appreciated.  

Yazan
yazan.soofi@gmail.com
(313)-410-0362

**Comment by MAKOMO** [comment]
We do not have much information on those machines. Do you have a valid network connection to the machine? Is your PC and your Roaster on the same network or are using a manual IP setup? Does your roasting machine and your PC have a different IP address on the same network assigned? Did you put the roasters IP address in the MODBUS tab (menu Config Ports >> 3rd tab MODBUS) under host?
